external_executors/local_cromwell/local_cromwell_template.json

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
I haven't done enough coding of this type to provide a good criticism of this document; you might want to get another set of eyes here.